Umee-Bots/lendingPool-reentrancy/src/agent.config.ts

```diff
@@ -0,0 +1,12 @@
+import utils, { AgentConfig } from "./utils";
+
+const CONFIG: AgentConfig = {
+  reentrancyFunctionsSelectors: utils.REENTRANCY_FUNCTIONS_SELECTORS,
```

Comment:
Maybe you could make a `reentrancyBlacklist` field instead (I think it's more descriptive than `reentrancyFunctions`) so that the client can input a list of signatures, like this:
```
{
  reentrancyBlacklist: [
    "signature()",
    // [...]
  ],
  // [...]
}
```

Then, inside `provideHandleTransaction`, you could compute the sighashes, like this:
```TS
export const handleTransaction = (config: AgentConfig): HandleTransaction => {
  const sighashes = getSighashes(config.reentrancyBlacklist);
  return async (txEvent: TransactionEvent): Finding[] => {
    // [...]
  }
```

Reply:
This would make it better, at least in a way, because the configuration would be centralized in `agent.config.ts`. If you do make these changes, you can also remove `REENTRANCY_FUNCTIONS_SIGNATURES` and `REENTRANCY_FUNCTIONS_ABI`, since they won't be used anymore.
Also, `getSighashes` would do something similar to what's being done currently - get `reentrancyBlacklist`, create an Interface from it and get the sighashes of all the functions.